javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -1174,19 +1179,9 @@ public static boolean isEnumSuper(MethodInvocationTree node) {
      * @return true if the tree is a type declaration
      */
     public static boolean isTypeDeclaration(Tree node) {
-        switch (node.getKind()) {
-                // These tree kinds are always declarations.  Uses of the declared
-                // types have tree kind IDENTIFIER.
-            case ANNOTATION_TYPE:
-            case CLASS:
-            case ENUM:
-            case INTERFACE:
-            case TYPE_PARAMETER:
-                return true;
-
-            default:
-                return false;
-        }
+        // These tree kinds are always declarations.  Uses of the declared types
```

Comment:
You removed this comment in the `Element` version of this method. I agree that the comment doesn't add anything useful, so I would be fine with removing it here too.